atlasdb-api/src/test/java/com/palantir/atlasdb/keyvalue/api/CellTest.java

```diff
@@ -80,6 +80,18 @@ public void testHashCode() {
                 .hasSameHashCodeAs(Cell.create(bytes("col"), bytes("row")));
     }
 
+    @Test
+    public void testSizeInBytes() {
+        assertThat(createCellWithByteSize(2).sizeInBytes()).isEqualTo(2);
+        assertThat(createCellWithByteSize(63).sizeInBytes()).isEqualTo(63);
+        assertThat(createCellWithByteSize(256).sizeInBytes()).isEqualTo(256);
+    }
+
+    private static Cell createCellWithByteSize(int size) {
+        Preconditions.checkArgument(size > 1, "Size should be at least 2");
```

Comment:
nit: since the assertion is at least 2, no need to rely on the discrete domain - just write `size >= 2`?